privilege/privileges/privileges.go

```diff
@@ -136,31 +135,15 @@ func (p *UserPrivileges) RequestVerification(activeRoles []*auth.RoleIdentity, d
 		}
 	}
 
-	switch dbLowerName {
-	case util.InformationSchemaName.L:
+	if util.IsMemDB(dbLowerName) {
```

Comment:
This PR also disallows creating views on `performance_schema` and `metrics_schema`.

Reply:
MySQL cannot create views in performance_schema.

```sql
mysql> create view performance_schema.v as select * from test.t;
ERROR 1044 (42000): Access denied for user 'root'@'%' to database 'performance_schema'
```